### ISSUE TITLE
Apply for funding (GSS, SPICES)

### ISSUE THREAD

**Issue by BrunoGrandePhD** [sciprog-sfu/sciprog-sfu.github.io#21]
We could try applying for funding from the GSS. They have [Grants for Caucuses](http://sfugradsociety.ca/?page_id=281) that we could apply for. I'm the president now of the MBB Grad Caucus, so I can bring this up at our next caucus meeting. Even thought MBB would be applying for the funding, the event would remain department-agnostic. We could use this funding for food, room booking (_e.g._ IRMACS, if they have nice rooms), etc.

**Comment by aliojjati** [comment]
I can talk to the Physics Caucus if that is useful ...

**Comment by BrunoGrandePhD** [comment]
We'll see if that's necessary. I doubt GSS would allow obtaining funding twice for the same event. We'll see. On the other hand, once more people fill the survey, we'll have a better idea of demographics and ask the departments and faculties for funding proportional to the number of students attending our events. 

This might require budgeting to see what we would use the money for. One way to get students to come is free food. Imagine having pizza during the workshop. Just an idea. 

@BillMills, do you have any food at the study group events at UBC?

**Comment by katieannemills** [comment]
@brunogrande nope, the UBC group runs exclusively on good vibes - but, the Toronto group is in the process of formalizing their existence in a way that gives them access to funds etc; you can check out that conversation [here](https://github.com/UofTCoders/studyGroup/pull/20).

**Comment by rchurchley** [comment]
Math Grad Caucus secretary and GSS Councillor here. I happened to be passing by the GSS office today so I stopped in to ask about funding. They confirmed that @brunogrande's suggestion of applying for a Caucus Grant is our best bet.

The way Caucus Grants currently work is that one caucus (MBB?) has to apply for the funding and commit some portion of its core operating funds towards the event. It isn't necessary for other caucuses to apply for the same grant (only one of the applications would be funded anyways). However, it might be useful for other caucuses to support the workshops by committing some of their core funding as well. I think a budget will be a prerequisite to asking for money, though.

For now, I'd suggest that it would be useful for us to go to our respective caucuses and just promote the study group. Then, once we have a handle on the budget, we can ask them for some fair amount of funding from each caucus.

**Comment by BrunoGrandePhD** [comment]
@rchurchley, thanks a lot for looking into this! This information will allow us to plan things accordingly. 

I agree on the point that we should first focus on promoting the study group and building a community. This should allow for regular attendance to the sessions. In turn, this would simplify the process of budgeting, something that would be required by most funding sources. Furthermore, we could ask the community what it would like as opposed to guessing.

**Comment by BrunoGrandePhD** [comment]
I just remembered another funding opportunity that we could apply for (and not be tied to a specific caucus). http://www.sfu.ca/dean-gradstudies/events/SPICES.html

Again, once we have a budget of what we could use this for, we should look into applying.

**Comment by radaniba** [comment]
@brunogrande sorry I just pulled that discussion now, why are you applying for funding at first place ? Did you think about CrowdFunding ? That can be a good exercice !

**Comment by BrunoGrandePhD** [comment]
We haven't really discussed this in depth yet. However, it could help with booking better rooms and perhaps have food (like pizza during workshops or help pay off hacky hours). 

Crowdfunding is definitely an option, but I don't know about asking for money from the community who would benefit from it. It would feel like they're paying for their admittance. And if not everyone joins in, then some would be paying for the benefit of others. What do you think?

**Comment by radaniba** [comment]
There are different types of crowdfunding, what you see in let's say `kickstarter` and what you see in `tilt`
In the first there are benefits back to contributors, depending on the amount given. In the second, you more rely on people 'kindness' :) where community member don't expect anything back (https://www.tilt.com/learn) , since we are not designing a product here, the second category is more appropriate

**Comment by rchurchley** [comment]
It would probably best for a rough budget to come first before we discuss funding sources in great detail. I'm expecting we'll only need a small amount of money in the grand scheme of things, and it's probable that individual caucus funding, GSS Caucus grant funding, and SPICES will provide more than enough money before launching a crowdfunding campaign. (In fact, if we obtained and exhausted those funding sources within the first year of the study group, I'd consider it a sign that we were spending too much money too quickly!)

I suppose the question is open as to whether we should _prefer_ crowdfunding to grant funding, but as far as I know, the grants don't come with any onerous strings attached. I don't see any reason not to apply for them first before asking for money from the community.
